spec/unit_tests/moab/storage_services_spec.rb

```diff
@@ -3,6 +3,21 @@
 describe Moab::StorageServices do
   let(:eq_xml_opts) { { element_order: false, normalize_whitespace: true } }
 
+  it 'only instantiates StorageRepository once for the class, even if multiple threads try to access it' do
+    # Object#object_id Returns an integer identifier for obj.  The same number will be returned
+    # on all calls to object_id for a given object, and no two active objects will share an id.
+    # https://ruby-doc.org/core-3.0.1/Object.html#method-i-object_id
```

Comment:
perhaps an unnecessary comment, can remove if people feel it's overkill or distraction